extensions/wikidata/module/scripts/menu-bar-extension.js

```diff
@@ -1,134 +1,153 @@
-		dictionary = data['dictionary'];
-		lang = data['lang'];
-	}
+  },
+  success: function (data) {
+    dictionary = data['dictionary'];
+    lang = data['lang'];
+  }
 });
 $.i18n().load(dictionary, lang);
 
 
-
 ExporterManager.MenuItems.push({});
-ExporterManager.MenuItems.push(
-        {
-            id:"performWikibaseEdits",
-            label: $.i18n('wikidata-extension/wikidata-edits'),
-            click: function() { PerformEditsDialog.checkAndLaunch(); }
-        });
-ExporterManager.MenuItems.push(
-        {               
-            id:"exportQuickStatements",
-            label: $.i18n('wikidata-extension/qs-file'),
-            click: function() { WikibaseExporterMenuBar.checkSchemaAndExport("quickstatements"); }
-        });
-ExporterManager.MenuItems.push(
-        {               
-            id:"exportWikibaseSchema",
-            label: $.i18n('wikidata-extension/wikidata-schema'),
-            click: function() { WikibaseExporterMenuBar.checkSchemaAndExport("wikibase-schema"); }
-        }
-);
+ExporterManager.MenuItems.push({
+  id: "performWikibaseEdits",
+  label: $.i18n('wikibase-extension/wikibase-edits'),
+  click: function () {
+    PerformEditsDialog.checkAndLaunch();
+  }
+});
+ExporterManager.MenuItems.push({
+  id: "exportQuickStatements",
+  label: $.i18n('wikibase-extension/qs-file'),
+  click: function () {
+    WikibaseExporterMenuBar.checkSchemaAndExport("quickstatements");
+  }
+});
+ExporterManager.MenuItems.push({
+  id: "exportWikibaseSchema",
+  label: $.i18n('wikibase-extension/wikibase-schema'),
+  click: function () {
+    WikibaseExporterMenuBar.checkSchemaAndExport("wikibase-schema");
+  }
+});
 
 WikibaseExporterMenuBar = {};
 
-WikibaseExporterMenuBar.exportTo = function(format) {
-    var targetUrl = null;
-    if (format ==="quickstatements") {
-        targetUrl = "statements.txt";
-    } else {
-        targetUrl = "schema.json";
-    }
-    var form = document.createElement("form");
-    $(form).css("display", "none")
-        .attr("method", "post")
-        .attr("action", "command/core/export-rows/"+targetUrl)
-        .attr("target", "gridworks-export-"+format);
-    $('<input />')
-        .attr("name", "engine")
-        .attr("value", JSON.stringify(ui.browsingEngine.getJSON()))
-        .appendTo(form);
-    $('<input />')
-        .attr("name", "project")
-        .attr("value", theProject.id)
-        .appendTo(form);
-    $('<input />')
-        .attr("name", "format")
-        .attr("value", format)
-        .appendTo(form);
+WikibaseExporterMenuBar.exportTo = function (format) {
+  var targetUrl = null;
+  if (format === "quickstatements") {
+    targetUrl = "statements.txt";
+  } else {
+    targetUrl = "schema.json";
+  }
+  var form = document.createElement("form");
+  $(form).css("display", "none")
+      .attr("method", "post")
+      .attr("action", "command/core/export-rows/" + targetUrl)
+      .attr("target", "gridworks-export-" + format);
```

Comment:
"Gridworks" is the name that preceded "Google Refine", which was replaced by "OpenRefine" so occurrences of "gridworks" should be globally replaced with "openrefine"